openlibrary/plugins/worksearch/schemes/lists.py

```diff
@@ -0,0 +1,43 @@
+from datetime import datetime
+import logging
+from collections.abc import Callable
+
+from openlibrary.plugins.worksearch.schemes import SearchScheme
+
+logger = logging.getLogger("openlibrary.worksearch")
+
+
+class ListSearchScheme(SearchScheme):
+    universe = ['type:list']
+    all_fields = {  # I have no idea what else to add.
+        'key',
+        'name',
```

Comment:
Add:
- seed
-         "subject",
        "person",
        "place",
        "time",
-         "subject_key",
        "person_key",
        "place_key",
        "time_key",

Reply:
Done. Btw, how did you figured this part out?